app/editor/src/features/admin/topics/constants/columns.tsx

```diff
@@ -1,36 +1,110 @@
-import { CellCheckbox, CellEllipsis, ITableHookColumn, ITopicModel } from 'tno-core';
-
-export const columns: ITableHookColumn<ITopicModel>[] = [
-  {
-    label: 'Name',
-    name: 'name',
-    width: 3,
-    cell: (cell) => <CellEllipsis>{cell.original.name}</CellEllipsis>,
-  },
-  {
-    label: 'Description',
-    name: 'description',
-    width: 3,
-    cell: (cell) => <CellEllipsis>{cell.original.description}</CellEllipsis>,
-  },
-  {
-    label: 'Type',
-    name: 'topicType',
-    width: 1,
-    cell: (cell) => <CellEllipsis>{cell.original.topicType}</CellEllipsis>,
-  },
-  {
-    label: 'Order',
-    name: 'sortOrder',
-    width: 1,
-    hAlign: 'center',
-    cell: (cell) => cell.original.sortOrder,
-  },
-  {
-    label: 'Enabled',
-    name: 'isEnabled',
-    width: 1,
-    hAlign: 'center',
-    cell: (cell) => <CellCheckbox checked={cell.original.isEnabled} />,
-  },
-];
+import React from 'react';
+import {
+  Button,
+  ButtonVariant,
+  FieldSize,
+  getEnumStringOptions,
+  ITableHookColumn,
+  ITopicModel,
+  OptionItem,
+  Select,
+  TopicTypeName,
+} from 'tno-core';
+
+export const Columns = (
```

Comment:
The coding standard for javascript is to use capital letters for classes and components.  This is a function and should use the standard camel case format.

Reply:
Updated to a component as using React hooks here, thanks